src/serviceregistry/manual/examples.md

```diff
@@ -0,0 +1,2390 @@
+        "metadata": {
+          "type": "temperature",
+          "customizable": true,
+          "scales": ["Fahrenheit", "Kelvin", "Celsius"]
+        },
+        "version": "2",
+        "addresses": [
+          {
+            "type": "IPV4",
+            "address": "128.0.1.2"
+          },
+          {
+            "type": "MAC",
+            "address": "12:34:5a:ff:c9:43"
+          }
+        ],
+        "deviceName": "thermometer5"
+      }
+    ]
+  }
+  ~~~
+
+  Response body:
+  ~~~
+  {
+    "entries": [
+      {
+        "name": "test-temperature-provider",
+        "metadata": {
+          "type": "temperature",
+          "scales": []
+        },
+        "version": "1.0.0",
+        "addresses": [
+          {
+            "type": "MAC",
+            "address": "00:00:00:00:00:01"
+          }
+        ],
+        "createdAt": "2024-10-23T21:43:34.863607800Z",
+        "updatedAt": "2024-10-23T21:43:34.863607800Z"
+      },
+      {
+        "name": "main-temperature-provider",
+        "metadata": {
+          "type": "temperature",
+          "customizable": true,
+          "scales": [
+            "Fahrenheit",
+            "Kelvin",
+            "Celsius"
+          ]
+        },
+        "version": "2.0.0",
+        "addresses": [
+          {
+            "type": "IPV4",
+            "address": "128.0.1.2"
+          },
+          {
+            "type": "MAC",
+            "address": "12:34:5a:ff:c9:43"
+          }
+        ],
+        "device": {
+          "name": "thermometer5",
+          "metadata": {
+            "scales": [
+              "Fahrenheit",
+              "Kelvin",
+              "Celsius"
+            ],
+            "max-temperature": {
+              "Kelvin": 310,
+              "Celsius": 40,
+              "Fahrenheit": 140
+            },
+            "min-temperature": {
+              "Fahrenheit": 20,
+              "Kelvin": 260,
+              "Celsius": -10
+            },
+            "appliable": [
+              "spring",
+              "summer",
+              "autumn",
+              "winter"
+            ]
+          },
+          "addresses": [
+            {
+              "type": "MAC",
+              "address": "81:ef:1a:44:7b:02"
+            }
+          ],
+          "createdAt": "2024-10-22T10:34:10Z",
+          "updatedAt": "2024-10-22T10:34:10Z"
+        },
+        "createdAt": "2024-10-23T21:43:34.869614800Z",
+        "updatedAt": "2024-10-23T21:43:34.869614800Z"
+      }
+    ],
+    "count": 2
+  }
+  ~~~
+
+  - **Update systems:** PUT /serviceregistry/mgmt/systems
+  
+  Request body:
+  ~~~
+  {
+    "systems": [
+      {
+        "name": "temperature-consumer",
+        "metadata": {
+          "type": "temperature",
+          "screen-size": {"width": 600, "height": 350}
+        },
+        "version": "2.2.1",
+        "addresses": [
+          {
+            "type": "MAC",
+            "address": "2a:cc:67:89:5f:ff"
+          }
+        ],
```

Comment:
- only string list
- recommended not to use MAC address with systems, you can use hostname instead